GameBoard/Views/GameEventParticipation/RemoveFromGameEvent.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model GameBoard.Models.GameEvent.RemoveFromGameEventViewModel
+
+<article>
+    <h1>Removing @Model.UserName.</h1>
+    <p>You're about to remove @Model.UserName from the @Model.GameEventName event. You will be able to send this user an invitation to this event later. Are you sure?</p>
```

Comment:
`"(...) You will be able to invite him again later (...)" would sound better IMO.

Reply:
Fixed.